internal/home/web.go

```diff
@@ -57,6 +57,9 @@ type webConfig struct {
 	firstRun bool
 
 	serveHTTP3 bool
+
+	// ciphers specified by user
+	tlsCiphers []string
```

Comment:
It's better to [parse structured data as soon as possible](https://lexi-lambda.github.io/blog/2019/11/05/parse-don-t-validate/), so it's better to have `[]uint16` here as well.

Also, the documentation should be improved, like in the other cases, and the slice should be moved higher to prevent the structure from having suboptimal field alignment.

Reply:
changes done